packages/@expo/cli/src/lint/lintAsync.ts

```diff
@@ -0,0 +1,115 @@
+import JsonFile from '@expo/json-file';
+import * as PackageManager from '@expo/package-manager';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs/promises';
+import path from 'path';
+
+import { Log } from '../log';
+import { selectAsync } from '../utils/prompts';
+
+const WITH_PRETTIER = `module.exports = {
+  root: true,
+  extends: ["universe/native", "prettier"],
+  plugins: ["prettier", "expo"],
+  rules: {
+    "prettier/prettier": ["warn"],
+    "expo/no-env-var-destructuring": ["error"],
+    "expo/no-dynamic-env-var": ["error"],
+  },
+};
+
+`;
+
+const ESLINT_ONLY = `module.exports = {
+  root: true,
+  extends: ["universe/native"],
+  plugins: ["expo"],
+  rules: {
+    "expo/no-env-var-destructuring": ["error"],
+    "expo/no-dynamic-env-var": ["error"],
+  },
+};
+`;
+
+const setupLinting = async (projectRoot: string) => {
+  const result = await selectAsync(
+    'No eslint config found. Would you like to set up linting for this project?',
+    [
+      {
+        title: 'Yes, eslint only',
+        value: 'eslint',
+      },
+      {
+        title: 'Yes, eslint and prettier',
+        value: 'eslint-and-prettier',
+      },
+      {
+        title: 'No',
+        value: 'no',
+      },
+    ]
+  );
+
+  if (result === 'no') {
+    return;
+  }
+
+  const commandSegments = [
+    'expo',
+    'install',
+    'eslint',
+    'eslint-plugin-expo',
+    'eslint-config-universe', // TODO(Kadi): eslit-config-expo once published
+  ];
+
+  if (result === 'eslint-and-prettier') {
+    commandSegments.push('prettier');
+    commandSegments.push('eslint-config-prettier');
+    commandSegments.push('eslint-plugin-prettier');
+  }
+
+  // TODO(Kadi): how to add this as dev dependencies?
+  await spawnAsync('npx', commandSegments, {
+    stdio: 'inherit',
+    cwd: projectRoot,
+    env: { ...process.env },
+  });
```

Comment:
Will we ever list `eslint`, `eslint-plugin-expo`, and/or prettier plugins as versioned dependencies? cc @brentvatne

If not, you can simplify this with just:

```ts
const manager = PackageManager.createForProject(projectRoot);
await manager.addAsync(['eslint', 'eslint-plugin-expo', ...]);
```

Reply:
@byCedric damn, I missed that! Thanks I knew there must have been an easier way to do it. Any chance you know how to make these into _dev_ dependencies? I couldn't see an option for it.